service-api/src/main/java/greencity/client/UserRemoteClient.java

```diff
@@ -111,6 +112,14 @@ public interface UserRemoteClient {
      * @param employeeEmail    {@link String}
      * @param newEmployeeEmail {@link String}
      */
-    @PutMapping("user/update-employee-email")
+    @PutMapping("/user/update-employee-email")
```

Comment:
Please follow the name convention of API according to the REST standard.

Reply:
done, i deleted the update from url, because I hane not to show which crud operation I use

Reply:
done, i deleted the update from url, because I hane not to show which crud operation I use